opentelemetry-api/tests/baggage/test_baggage_propagation.py

```diff
@@ -26,14 +27,17 @@ def get_as_list(
     return dict_object.get(key, [])
 
 
+getter = HelperGetter(get_as_list, DictGetter.keys)
```

Comment:
I think we should use DictGetter directly for such cases. We just need to modify DictGetter to work with both of the following types of dictionaries:

1. `Dict[str, str]`
2. `Dict[str, Collection[str]]`

DictGetter should work with both of these cases. Once we support that, we can use dict getter directly in places like these.

Reply:
I've added this removed duplicate get implementations in other files